app/assets/javascripts/oxalis/constants.js

```diff
@@ -91,6 +91,13 @@ export type ContourModeType = $Keys<typeof ContourModeEnum>;
 export const NODE_ID_REF_REGEX = /#([0-9]+)/g;
 export const POSITION_REF_REGEX = /#\(([0-9]+,[0-9]+,[0-9]+)\)/g;
 
+// The plane in orthogonal mode is a little smaller than the viewport
+// as there are two borders with width 2px each => 8px
+// TODO: Wouldn't 2px on each side only result in 4px additional width instead of 8px?
```

Comment:
There are 2 borders, like this: `||...data...||`, the yellow and the red border for example, each border is 2px in width, so 2*2px on the left and 2*2px on the right, making 8px.

Reply:
Ahh, ok, I thought the input catcher border wouldn't be counted and misread the "two" in the sense of left and right. Thanks for the clarification :+1: